locales/en/tasks.json

```diff
@@ -62,7 +62,7 @@
     "perfectText": "Completed all active Dailies on <%= perfects %> days. With this achievement you get a +level/2 buff to all attributes for the next day.",
     "perfectSingular": "Perfect Day",
     "perfectSingularText": "Completed all active Dailies in one day. With this achievement you get a +level/2 buff to all attributes for the next day.",
-    "streakerAchievement": "You have attained the \"Streaker\" Achievement! The 21-day mark is a milestone for habit formation. You can continue to stack this Achievement for every additional 21 days, on this Daily or any other!",
+    "streakerAchievement": "You have attained the \"Streaker\" Achievement! The 21-day mark is a milestone for Habit formation. You can continue to stack this Achievement for every additional 21 days, on this Daily or any other!",
```

Comment:
I agree with all of Blade's assessments. (I think "Habits to be done"
should be capitalized.)

On Wed, Jan 28, 2015 at 7:25 AM, Blade Barringer <notifications@github.com>
wrote:

> In locales/en/tasks.json
> <https://github.com/HabitRPG/habitrpg-shared/pull/456#discussion_r23694032>
> :
>
> > @@ -62,7 +62,7 @@
> >      "perfectText": "Completed all active Dailies on <%= perfects %> days. With this achievement you get a +level/2 buff to all attributes for the next day.",
> >      "perfectSingular": "Perfect Day",
> >      "perfectSingularText": "Completed all active Dailies in one day. With this achievement you get a +level/2 buff to all attributes for the next day.",
> > -    "streakerAchievement": "You have attained the \"Streaker\" Achievement! The 21-day mark is a milestone for habit formation. You can continue to stack this Achievement for every additional 21 days, on this Daily or any other!",
> > +    "streakerAchievement": "You have attained the \"Streaker\" Achievement! The 21-day mark is a milestone for Habit formation. You can continue to stack this Achievement for every additional 21 days, on this Daily or any other!",
>
> Talking generally about habits again, lowercase.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg-shared/pull/456/files#r23694032>.
>